src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -308,6 +309,18 @@ private void InitDiscovery()
         SameKeyGenerator privateKeyProvider = new(_api.NodeKey.Unprotect());
         NodeIdResolver nodeIdResolver = new(_api.EthereumEcdsa);
 
+        if (discoveryConfig.Discv5Enabled)
+        {
+            SimpleFilePublicKeyDb discv5DiscoveryDb = new(
+                "EnrDiscoveryDB",
+                DiscoveryNodesDbPath.GetApplicationResourcePath(_api.Config<IInitConfig>().BaseDbPath),
+                _api.LogManager);
+
+            _api.DiscoveryApp = new DiscoveryV5App(privateKeyProvider, _api, _networkConfig, discoveryConfig, discv5DiscoveryDb, _api.LogManager);
+            _api.DiscoveryApp.Initialize(_api.NodeKey.PublicKey);
+            return;
```

Comment:
Just to confirm - all the steps below are only for V4?

Reply:
Yep